SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md

```diff
@@ -0,0 +1,75 @@
+<%(TOC)%>
+# Multiserver Shooter Tutorial
+
+## Step 4: Test your changes in the cloud
+
+### Build your assemblies
+
+An assembly is what’s created when you run `BuildWorker.bat`. They’re .zip files that contains all the files that your game uses when running in the cloud.
+
+1. In a terminal window, change directory to the root directory of the Third Person Shooter repository.
+2. Build a server-worker assembly by running: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat ThirdPersonShooterServer Linux Development ThirdPersonShooter.uproject`
+3. Build a client-worker assembly by running: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat ThirdPersonShooter Win64 Development ThirdPersonShooter.uproject`
+
+<br/>
+### Upload your game
+
+1. In File Explorer, navigate to `UnrealGDKThirdPersonShooter\spatial` and open `spatialos.json` in a text editor.
+2. Change the `name` field to the name of your project. You can find this in the [Console](https://console.improbable.io). It’ll be something like `beta_someword_anotherword_000`.
+    ![]({{assetRoot}}assets/tutorial/project-name.png)
+3. In a terminal window, change directory to `UnrealGDKThirdPersonShooter\spatial\` and run `spatial cloud upload <assembly_name>`, where `<assembly_name>` is a name of your choice (for example `myassembly`). A valid upload command looks like this:
+
+```
+spatial cloud upload myassembly
+```
+
+> **Note:** Based on your network speed it may take a little while (1-10 minutes) to upload your assembly.
```

Comment:
```suggestion
> **Note:** Depending on your network speed it may take a little while (1-10 minutes) to upload your assembly.
```
Also, if we can use fancy callouts, could change this to be a "tip" or "info" box?